File: CienciasDaComputação/Maratona SBC/Testes/Diferenca/diferenca.c
#include <stdio.h>

int main(){
    int A = 0, B = 0, C = 0, D = 0;
    scanf("%d %d %d %d", &A, &B, &C, &D);
    int DIFERENCA = (A * B - C * D);
    printf("DIFERENCA = %d\n", DIFERENCA);
    return 0;
}

File: CienciasDaComputação/Maratona SBC/Testes/Diferenca/diferenca.py
A = B = C = D = 0   # todas começam com 0
A = int(input())
B = int(input())
C = int(input())
D = int(input())
DIFERENCA = (A * B - C * D)
print("DIFERENCA = " , DIFERENCA)

A = B = C = D = 0   
A, B, C, D = list(map(int, input().split()))
DIFERENCA = (A * B - C * D)
print("DIFERENCA = " , DIFERENCA)